loader/src/ui/nodes/TextInput.cpp

```diff
@@ -70,7 +70,7 @@ bool TextInput::init(float width, std::string const& placeholder, std::string co
     m_bgSprite->setContentSize({ width * 2, HEIGHT * 2 });
     this->addChildAtPosition(m_bgSprite, Anchor::Center);
 
-    m_input = CCTextInputNode::create(width, HEIGHT, placeholder.c_str(), 24, font.c_str());
+    m_input = CCTextInputNode::create(width-10.f, HEIGHT, placeholder.c_str(), 24, font.c_str());

```

Comment:
can you add a space around the -  this is very important

Reply:
true